README.md

```diff
@@ -1,2 +1,8 @@
 # rosvr
 RedHat Open Source Virtual Reality
+
+# Requirements
+```
+yum install node
+npm install express body-parser path serve-index
```

Comment:
you can use package.json for asset management:
https://docs.npmjs.com/files/package.json#dependencies